build.xml

```diff
@@ -260,4 +261,53 @@
         </exec>
     </target>
 
-</project>
\ No newline at end of file
+    <target name="composer-check-dependencies" description="Checks transcendent dependencies.">
```

Comment:
You misused the word "transcendent", you should probably use the word "transitive" instead.

**transcendent**
_/tranˈsɛnd(ə)nt,ˌtrɑːnˈsɛnd(ə)nt/_
adjective
1. beyond or above the range of normal or physical human experience.
"the search for a transcendent level of knowledge"
2. (in scholastic philosophy) higher than or not included in any of Aristotle's ten categories.

**transitive**
_/ˈtranzɪtɪv,ˈtrɑːnzɪtɪv,ˈtransɪtɪv,ˈtrɑːnsɪtɪv/_
adjective
1. GRAMMAR (of a verb or a sense or use of a verb) able to take a direct object (expressed or implied), e.g. saw in he saw the donkey.
2. LOGIC/MATHEMATICS (of a relation) such that, if it applies between successive members of a sequence, it must also apply between any two members taken in order. For instance, if A is larger than B, and B is larger than C, then A is larger than C.